NaReguaApp/src/routes/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+
+import AuthContext from '../contexts/auth';
+
+import AppClienteRoute from './app-cliente.routes';
+import AppPrestadorRoute from './app-prestator.routes';
+import AuthRoute from './auth.routes';
+
+function Routes() {
+	const { logado, isProvider } = useContext(AuthContext);
+
+	return (
+		<>
+			{logado ?
+				<>
+					{isProvider ? <AppClienteRoute /> : <AppPrestadorRoute />}
+				</>
+				: <AuthRoute />
+			}
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹↹` with `······)`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
      )}
```